packages/fuels/tests/contracts/assert_eq/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "assert_eq"
+
+[dependencies]
+std = { path = "../../../../../../sway/sway-lib-std" }
```

Comment:
This might be left over from debugging. Or are you waiting on a release? Not sure if I saw the CI pass :D

Reply:
Yes :) We need a new release